utils/h2ph.PL

```diff
@@ -851,6 +877,37 @@ DEFINE
 		    quotemeta($define{$_}), "\" } }\n\n";
 	    }
 	}
+
+	print PREAMBLE <<SIZEOF;
+%sizeof = (
+	char	=> $Config{charsize},
+	uchar	=> $Config{charsize},
+	u_char	=> $Config{charsize},
```

Comment:
The Windows failures are because `$Config{charsize}` isn't defined on Windows.

But these are always 1 (otherwise it's not even vaguely a C compiler), so these could just be set to 1